src/Themes/NexusMods.Themes.NexusFluentDark/NexusMods.Themes.NexusFluentDark.csproj

```diff
@@ -57,4 +57,19 @@
       <ProjectReference Include="..\..\NexusMods.Icons\NexusMods.Icons.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <AvaloniaXaml Update="Styles\Controls\ComboBox\ComboBox.AddPanelDropDown.axaml">
+        <DependentUpon>ComboBox.axaml</DependentUpon>
+      </AvaloniaXaml>
+      <AvaloniaXaml Update="Styles\Controls\ComboBox\ComboBox.Primary.axaml">
+        <DependentUpon>ComboBox.axaml</DependentUpon>
+      </AvaloniaXaml>
+      <AvaloniaXaml Update="Styles\Controls\ComboBox\ComboBox.Secondary.axaml">
+        <DependentUpon>ComboBox.axaml</DependentUpon>
+      </AvaloniaXaml>
+      <AvaloniaXaml Update="Styles\Controls\ComboBox\ComboBox.Tertiary.axaml">
+        <DependentUpon>ComboBox.axaml</DependentUpon>
+      </AvaloniaXaml>
+    </ItemGroup>
```

Comment:
I don't like all these separate files, especially if most of them share the same style but change the color.

Style files should also contain "Style" in the file name so that they can be distinguished from user controls during quick file search.  

I would put Primary, Secondary, Tertiary and ComboBox in a single file, up to you if you want to get ComboBoxItem in there as well but at least for that one it makes sense to have it separate.